Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workspace/TextFile.java

```diff
@@ -72,4 +73,12 @@ public void setRefId(String refId) {
         this.refId = refId;
     }
 
+    public String getTextFileURI() {
+        return TextFileURI;
+    }
+
+    public void setTextFileURI(String textFileURI) {
```

Comment:
This is not a good parameter name. It implies that I give the final file URI to the method, but that's not what the method needs. Especially with method names like this (getter/setter), you need to make very clear what the method expects. Othrewise, I assume that the default behavior applies. Also, make sure to write javadoc for this method in which you explain what it does!